python/ray/util/client/server/server.py

```diff
@@ -440,21 +441,26 @@ def PutObject(
         self, request: ray_client_pb2.PutRequest, context=None
     ) -> ray_client_pb2.PutResponse:
         """gRPC entrypoint for unary PutObject"""
-        return self._put_object(request, "", context)
+        return self._put_object(request.data, request.client_ref_id, "", context)
 
     def _put_object(
-        self, request: ray_client_pb2.PutRequest, client_id: str, context=None
+        self,
+        data: Union[bytes, bytearray],
```

Comment:
When will this be of `bytes` type?

Reply:
Technically if `PutObject` is called directly it will pass in `request.data` which would be `bytes`, although I think at the moment PutObject isn't call directly by anything.
```
    def PutObject(
        self, request: ray_client_pb2.PutRequest, context=None
    ) -> ray_client_pb2.PutResponse:
        """gRPC entrypoint for unary PutObject"""
        return self._put_object(request.data, request.client_ref_id, "", context)
```

Reply:
Ah! SGTM to leave as a union. Could you leave this as a comment?